assets/test/advanced_authoring/right_menu/screen/screen_mocks.ts

```diff
@@ -0,0 +1,66 @@
+import { IActivity } from "apps/delivery/store/features/activities/slice";
+
+export const screenSchema = {
```

Comment:
I think there are maybe really only 2 objects we want to mock actually, one is the actual screen data, this is what we transform into something that matches up with the schema in order to render the property fields. The other would be a "changes" object. We'd be simulating the changes being made on the edit form, and those are what get transformed  into model compatible format. we should probably have multiple variations of changes, simulating different things being "edited" and testing that the model version comes out of the method as expected. also it would be good to have a couple of palette versions to make sure that the conversion from legacy properties is happening